instrumentation/opentelemetry_phoenix/lib/opentelemetry_phoenix.ex

```diff
@@ -173,11 +173,6 @@ defmodule OpentelemetryPhoenix do
     |> OpenTelemetry.Ctx.attach()
   end
 
-  defp bandit_start() do
-    OpentelemetryProcessPropagator.fetch_parent_ctx()
-    |> OpenTelemetry.Ctx.attach()
-  end
```

Comment:
This function seems to have just been copied from the Cowboy adapter, but it breaks propagating the context for Bandit. This is because the telemetry events for Bandit are fired in the same process that handles the request (i.e., the same process that fires the Phoenix telemetry events). In effect, this was unsetting the already-set context.